src/blocks/general/PeopleHighlight/config.ts

```diff
@@ -0,0 +1,68 @@
+import type { Block } from 'payload';
+
+import { link } from '@/fields/link';
+
+export const PeopleHighlight: Block = {
+  slug: 'peopleHighlight',
+  interfaceName: 'PeopleHighlightBlock',
+  labels: {
+    singular: 'People highlight block',
+    plural: 'People highlight blocks',
+  },
+  fields: [
+    {
+      name: 'borderTop',
+      label: 'Border on top of block in desktop and tablet view',
+      type: 'checkbox',
+      defaultValue: false,
+    },
+    {
+      name: 'linkColor',
+      label: 'Link color',
+      type: 'select',
+      options: [
+        {
+          value: 'purple',
+          label: 'Purple',
+        },
+        {
+          value: 'red',
+          label: 'Red',
+        },
+        {
+          value: 'green',
+          label: 'Green',
+        },
+      ],
+    },
```

Comment:
Maybe it's time to break this out into a reusable function, since the three colors reoccur so often?